frontend/public/style/_overrides.scss

```diff
@@ -512,6 +512,10 @@ button.pf-c-dropdown__menu-item.pf-m-disabled {
   position: absolute !important;
 }
 
+#modal-container.pf-c-backdrop__open {
+  overflow: visible;
```

Comment:
Probably worth reaching out to the PatternFly team to:
1. determine why this declaration exists
1. let them know it is creating this issue